Issuetracker : #i56952#
CWS          : N/A
Author       : <maho@openoffice.org> (JCA)
Description  : FreeBSD porting : Passing $ORIGIN to linker at icu project
We also want to pass -rpath $ORIGIN to linker as GNU/Linux and/or Solaris's case.

--- icu/icu-2.6.patch	Fri Aug 26 20:43:35 2005
+++ icu-2.6.patch	Mon Oct 24 18:18:45 2005
@@ -2747,3 +2747,29 @@
       commands = pkg_appendToList(commands, NULL, uprv_strdup(stanza));
   
       sprintf(stanza, "@$(RMV) $(TEMP_DIR)/%s", cfile+genFileOffset);
+*** misc/icu/source/config/mh-bsd-gcc	Thu May  2 08:48:10 2002
+--- misc/build/icu/source/config/mh-bsd-gcc	Mon Oct 24 18:16:57 2005
+***************
+*** 10,21 ****
+--- 10,30 ----
+  GEN_DEPS.cc=	$(CXX) -E -MM $(DEFS) $(CPPFLAGS)
+  
+  ## Flags for position independent code
++ STATICCFLAGS = -fPIC 
++ STATICCXXFLAGS = -fPIC 
++ STATICCPPFLAGS = -DPIC 
+  SHAREDLIBCFLAGS = -fPIC
+  SHAREDLIBCXXFLAGS = -fPIC
+  SHAREDLIBCPPFLAGS = -DPIC
+  
+  ## Compiler switch to embed a runtime search path
+  LD_RPATH=	
++ ## Force RPATH=$ORIGIN to locate own dependencies w/o need for LD_LIBRARY_PATH
++ ENABLE_RPATH=YES
++ RPATHLDFLAGS=${LD_RPATH_PRE}'$$ORIGIN'
++ 
++ #SH#	ENABLE_RPATH=YES
++ #SH#	RPATHLDFLAGS="${LD_RPATH_PRE}'$$ORIGIN'"
+  LD_RPATH_PRE=	-Wl,-rpath,
+  
+  ## Compiler switch to embed a library name
